examples/opensora_hpcai/configs/opensora-v1-1/train/train_stage1.yaml

```diff
@@ -6,7 +6,7 @@ model_max_length: 200
 enable_flash_attention: True
 use_recompute: True
 
-vae_micro_batch_size: 4
+vae_micro_batch_size: 1
```

Comment:
when frame is large, e.g. 64, this will cause slower training time I think. Though for 16 frames there is no notificable difference

Reply:
Yes, but it will also allow larger buckets size. In this case, it's better to conduct performance measures and find a sweet spot between speed and batch size.